packages/secrets-manager/index.d.ts

```diff
@@ -1,5 +1,6 @@
 import { SecretsManager } from 'aws-sdk'
 import middy from '../core'
+import ssm from '../ssm'
```

Comment:
is this required? the two should be separate, right?

Reply:
Thanks for catching that! My IDE snuck that in ^^'
I'll remove it and double check the rest.

Reply:
I've removed the line